.gitignore

```diff
@@ -1,7 +1,5 @@
 /.vs/
 /Foreman/Foreman.csproj.user
-/Foreman/bin/
-/Foreman/obj/
-/ForemanTest/obj/
 /packages/
-/ForemanTest/bin/Release/
+[Bb]in/

```

Comment:
Please don't make such wide spread changes to the .gitignore. I'd rather see the originals kept as they target specific subdirectories. Was there a reason why this was necessary?

Reply:
I changed this to ignore files from ForemanTest/bin/Debug/